assets/src/components/ViewHeader.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react'
+import Typography from '@material-ui/core/Typography'
+
+function ViewHeader (props) {
+  const [domElement, setDomElement] = useState(null)
+  const { children } = props
+
+  // Focuses the page header only when the domElement changes.
+  // This should only be when it changes from null to something on initial render.
+  useEffect(() => {
+    if (domElement !== null) {
+      domElement.focus()
+    }
+  }, [domElement])
+
+  return (
+    <div tabIndex={0} ref={domElement => setDomElement(domElement)}>
```

Comment:
If we don't want this `div` to be in the Tab order, but there is some benefit when using a screen reader to focus on this element, we could consider changing this to `-1`. See [MDN's tabindex page](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).

Reply:
I've changed it to `-1` but also I'm removing the `tabIndex` when the focus moves away now.